pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> A tool to manage network booting of machines.
+> More information: <https://github.com/danderson/netboot/tree/master/pixiecore>.
+
+- Start a PXE boot server which provides a `netboot.xyz` boot image:
+
+`pixiecore {{quick}} {{xyz}} --dhcp-no-bind`
```

Comment:
If `xyz` is a dummy value, replace it with something else, because it's likely to be confused with `netboot.xyz`, where the `xyz` is **not** a dummy value.

Reply:
It's very much not a dummy value. This is precisely the line you are supposed to write. Does it mean that I used the `{{…}}` syntax wrong?

Reply:
That depends if it's the only value `pixiecore` accepts. Could you also specify a different server you want to get you image from?

Reply:
Oh, if you have to write `xyz`, then remove the token syntax (`{{}}`).  If it's one of a list of choices, then do `{{xyz|opt1|opt2|etc.}}`.

I'm guessing that the `xyz` is for `netboot.xyz`?

Reply:
Yes. For instance, here are a bunch of valid pixiecore commands:

```
pixiecore quick xyz
pixiecore quick ubuntu
pixiecore quick fedora
pixiecore boot <some more complicated but more flexible syntaxe>
pixiecode api <something even more complicated that I don't actually know anything about>
```

Reply:
@bl-ue it's one of a choice. But `xyz` (which, indeed, stands for netboot.xyz) is sort of the generic goto thing since it will let you chain into an arbitrary image after a computer PXE boots. So I think this particular command line is the one you'd want to appear for a quick copy-paste.

I'm being a bit thick, here, evidently, sorry about that. Hopefully we'll figure this out, though.

Reply:
How about we write the first example specifically for `netboot.xyz` and add a second one for `ubuntu` maybe just to show that it's possible.

Reply:
```suggestion
`pixiecore {{quick}} xyz --dhcp-no-bind`

- Start a new PXE boot server which provides a `ubuntu` boot image
```
(Or is it `an ubuntu`?)

Reply:
Also, is there maybe some kind of `pixiecore --list` command to get a list of all options for this?

Reply:
I think it's _an Ubuntu_.

`pixiecore  quick --help` gives the full list.

Reply:
Then we sould also include an example for that.

Reply:
So, maybe we'd like something like:
```suggestion
`pixiecore {{quick}} xyz --dhcp-no-bind`

- List predefined images:

`pixiecore quick --help`
```

Reply:
I applied your version. Made a tiny tweak to spell Ubuntu as a noun.